lib/api/src/externals/function.rs

```diff
@@ -431,11 +431,13 @@ impl Function {
         &self.store
     }
 
+    #[inline(always)]
     fn call_wasm(
         &self,
         trampoline: VMTrampoline,
         params: &[Val],
         results: &mut [Val],
+        unchecked: bool,
```

Comment:
If this function acts as a proxy for unchecked calling, it needs to be `unsafe` if we don't `catch_traps` ourselves inside this function